app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable,
          :confirmable, :masqueradable, :omniauthable
 
+  alias_method :is_confirmed?, :confirmed?
```

Comment:
removed duplicated LO method implementation for `is_confirmed` and aliases to the [version provided by Devise `confirmable`](https://github.com/plataformatec/devise/blob/master/lib/devise/models/confirmable.rb#L106-L108).

Reply:
Cool, I actually remove this is in one of my WIP PRs. But, I'd actually vote to remove the alias and just change all instances of `is_confirmed?` to `:confirmed`. What do you think?

Reply:
I agree it's better to nuke the java-esque doppleganger.. I'm was just nervous about updating a method name that follows a convention that (while discouraged in ruby) you or weston might have gotten used to. I'll remove it now